### ISSUE TITLE
App.js <a> Link Issue

### ISSUE THREAD

**Issue by Daniel-Peladeau** [Nicholas-Nazario/twitterLens#68]
./src/App.js
  Line 60:11:  The href attribute is required for an anchor to be keyboard accessible. Provide a valid, navigable address as the href value. If you cannot provide an href, but still need the element to resemble a link, use a button and change it with appropriate styles. Learn more: https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/anchor-is-valid.md  jsx-a11y/anchor-is-valid

**Comment by Daniel-Peladeau** [comment]
Same issue as GenericChart.js, adding an "href='# '" is the preferred fix.
